firmware/u2f.c

```diff
@@ -503,8 +503,15 @@ const HDNode *generateKeyHandle(const uint8_t app_id[], uint8_t key_handle[])
 const HDNode *validateKeyHandle(const uint8_t app_id[], const uint8_t key_handle[])
 {
 	uint32_t key_path[KEY_PATH_ENTRIES];
+	unsigned int i;
```

Comment:
We use C99 in most of the new code, so it is OK to declare loop variable in for cycle below.